fastlane/Fastfile

```diff
@@ -1,27 +1,29 @@
 default_platform(:android)
 fastlane_require 'dotenv'
 
+# NOTE: When updating the `promo_config` keys of those locale, ensure it matches the list
+# of locales enabled in `.circleci/config.yml` for the `raw-screenshots` job
 SUPPORTED_LOCALES = [
-  { glotpress: "ar", google_play: "ar",  promo_config: {}},
-  { glotpress: "de", google_play: "de-DE",  promo_config: {} },
+  { glotpress: "ar",    google_play: "ar",     promo_config: {} },
+  { glotpress: "de",    google_play: "de-DE",  promo_config: {} },
   { glotpress: "en-gb", google_play: "en-US",  promo_config: {} },
-  { glotpress: "es", google_play: "es-ES",  promo_config: {} },
-  { glotpress: "fr", google_play: "fr-CA",  promo_config: false },
-  { glotpress: "fr", google_play: "fr-FR",  promo_config: {} },
-  { glotpress: "he", google_play: "iw-IL",  promo_config: {} },
-  { glotpress: "id", google_play: "id",  promo_config: {} },
-  { glotpress: "it", google_play: "it-IT",  promo_config: {} },
-  { glotpress: "ja", google_play: "ja-JP",  promo_config: {} },
-  { glotpress: "ko", google_play: "ko-KR",  promo_config: {} },
-  { glotpress: "nl", google_play: "nl-NL",  promo_config: {} },
-  { glotpress: "pl", google_play: "pl-PL",  promo_config: {} },
+  { glotpress: "es",    google_play: "es-ES",  promo_config: {} },
+  { glotpress: "fr",    google_play: "fr-CA",  promo_config: {} },
```

Comment:
This one is not part of [the list of `locales:` parameter from circleci config](https://github.com/wordpress-mobile/WordPress-Android/pull/12869/files#diff-1d37e48f9ceff6d8030570cd36286a61R392) and should thus be excluded from promo screenshots
```suggestion
  { glotpress: "fr",    google_play: "fr-CA",  promo_config: false },
```

(Alternatively, maybe we could remove that line entirely to only keep `fr-FR`?)